meals/templates/meals_list.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base_table.html" %}
+
+{% load static %}
+{% load bootstrap3 %}
+
+{% block head_title %}Meals list{% endblock %}
+
+{% block extra_panel %}
+<a href="{% url 'meal_add' %}" class="btn btn-info" style="float: right;">Add new meal</a>
```

Comment:
Maybe move to footer block instead and btn-block (similar to https://github.com/hackupc/registration/blob/master/organizers/templates/invite_list.html#L9 )

Reply:
Noted on #52.

Reply:
Done!